### PR TITLE
feat: config react-query / subs fetch

### PR DESCRIPTION
**Pull request by Lucas-Onofre**
#31 - React Query configuration and addition of fetching the subscribers count using it.

### CODE REVIEW

**Comment by anti-duhring** [comment]
Nice! Can you solve the conflicts within `yarn.lock`?

**Review by ocodista — APPROVED** [review]
Outstanding!

Thank you!!!

File: package.json [hunk 1]
```diff
@@ -43,5 +43,8 @@
     "*.{json,md}": [
       "prettier --write"
     ]
+  },
+  "dependencies": {
+    "react-query": "^3.39.3"
   }
```
Comment (ocodista) [inline package.json:49]: This dependency is at the wrong package.json file.

You should add it on the `apps/web` page

**Review by ocodista — CHANGES_REQUESTED** [review]

**Comment by ocodista** [comment]
Close as I think it's already merged.

**Pull request closed without merging**